### ISSUE TITLE
[BUG] Github action doesn't report warnings

### ISSUE THREAD

**Issue by L3odr0id** [dart-code-checker/dart-code-metrics-action#142]
* **Dart Code Metrics GitHub Action version:** 3.0.0

**Please show your full configuration:**

<details>

<summary>CI configuration</summary>

```yaml
name: Dart Code Metrics

on: [pull_request, workflow_dispatch]

jobs:
  check:
    name: dart-code-metrics-action

    runs-on: ubuntu-latest

    steps:
      - uses: actions/checkout@v3

      - name: dart-code-metrics
        uses: dart-code-checker/dart-code-metrics-action@v3.0.0
        with:
          github_token: ${{ secrets.GITHUB_TOKEN }}
          pull_request_comment: true
          fatal_warnings: true
          fatal_performance: true
          fatal_style: true
          check_unused_files: false
```

</details>

<details>

<summary>analysis_options.yaml</summary>

```yaml
analyzer:
  ### extra_pedantic ###
  strong-mode:
      # https://github.com/dart-lang/language/tree/master/resources/type-system
      implicit-casts: false
      implicit-dynamic: true
  language:
    # https://github.com/dart-lang/language/tree/master/resources/type-system
    strict-inference: true
    strict-raw-types: true
    strict-casts: true
  errors:
    inference_failure_on_function_invocation: false
  ### extra_pedantic ###

  plugins:
    - dart_code_metrics
  exclude:
    - '**/*.g.dart'
    - '**/*.freezed.dart'
    - '**/*.gr.dart'
    - lib/generated/**

dart_code_metrics:
  metrics:
    cyclomatic-complexity: 10
    halstead-volume: 500
    lines-of-code: 80
    maintainability-index: 40
    maximum-nesting-level: 5
    number-of-methods: 10
    number-of-parameters: 8
  metrics-exclude:
    - test/**
    - lib/setup.dart
  rules:
    # Common rules
    - no-boolean-literal-compare
    - no-empty-block
    - prefer-trailing-comma
    - prefer-first
    - prefer-last
    - no-equal-then-else
    # Flutter specific
    - avoid-border-all
    - avoid-returning-widgets
    - avoid-unnecessary-setstate
    - avoid-wrapping-in-padding
    - prefer-const-border-radius
    - prefer-extracting-callbacks
    - prefer-single-widget-per-file:
        ignore-private-widgets: true
  anti-patterns:
    - long-method
    - long-parameter-list
   ```

</details>

**What did you expect to happen?**

When I run ` flutter pub run dart_code_metrics:metrics analyze lib` locally, the output shows me warnings and alarms as I expect. 

<details>

<summary>Part of the real local cli output</summary>

```
lib/core/theme/d3p_special_colors.dart:
WARNING CustomColors
        number of methods: 14 methods


lib/core/theme/d3p_special_styles.dart:
WARNING CustomTextStyles
        number of methods: 11 methods
```

</details>

I expect ci to show me the same warnings.

**What actually happened?**

Github actions ci retuns report with 0 issues.

<details>
<summary>Real github actions report</summary>

Report:
Summary
Scanned folders: lib

- Total scanned files: 227

- Total lines of source code: 3997

- Total classes: 264

- Average Cyclomatic Number per line of code: 0.23

- Average Source Lines of Code per method: 6

- Total tech debt: not found

- Found issues: 0 ✅

</details>

**Are you willing to submit a pull request to fix this bug?**
I think it's not a bug, but my mistake in configuration.
